superset-frontend/src/SqlLab/components/ExploreCtasResultsButton/index.tsx

```diff
@@ -17,36 +17,35 @@
  * under the License.
  */
 import React from 'react';
-import { useSelector } from 'react-redux';
-import { t } from '@superset-ui/core';
+import { useSelector, useDispatch } from 'react-redux';
+import { t, JsonObject } from '@superset-ui/core';
+import {
+  createCtasDatasource,
+  addInfoToast,
+  addDangerToast,
+} from 'src/SqlLab/actions/sqlLab';
 import { InfoTooltipWithTrigger } from '@superset-ui/chart-controls';
 import Button from 'src/components/Button';
 import { exploreChart } from 'src/explore/exploreUtils';
 import { SqlLabRootState } from 'src/SqlLab/types';
 
 interface ExploreCtasResultsButtonProps {
-  actions: {
-    createCtasDatasource: Function;
-    addInfoToast: Function;
-    addDangerToast: Function;
-  };
   table: string;
   schema?: string | null;
   dbId: number;
   templateParams?: string;
 }
 
 const ExploreCtasResultsButton = ({
-  actions,
   table,
   schema,
   dbId,
   templateParams,
 }: ExploreCtasResultsButtonProps) => {
-  const { createCtasDatasource, addInfoToast, addDangerToast } = actions;
   const errorMessage = useSelector(
     (state: SqlLabRootState) => state.sqlLab.errorMessage,
   );
+  const dispatch = useDispatch<(dispatch: any) => Promise<JsonObject>>();
```

Comment:
Define `any`

Reply:
Currently `any` is ok because `actions/sqlLab.js` is not typed

Reply:
I don't think we need to type `useDispatch` though - in most places we just do `const dispatch = useDispatch()`